### DISCUSSION TITLE
Noisy reactions

### DISCUSSION THREAD

**Discussion by max-baz** [isamert/scli#174] [Ideas]
I happen to receive lots of reactions in different chats, and it got me thinking about different ways we notify about them and whether it's useful...

First of all, in the bottom of `scli` we should information about a reaction, who did it and what was the reaction - I think it's useful, it's not distracting and it has some context 👍 

Then we count notifications in parenthesis in front of every chat - this I would say is not really useful, if I see `(23) Some Chat` and I open it and see no new messages, I'm a bit confused (I can infer that those 23 must have been reactions, but where they are I don't really know, nor do I probably care to track each one of them down 🙂 )

Finally we also have desktop notifications, which could be quite noisy, as reactions don't really require some _answer_ from me. 

It could be just my personal habits, that's why I thought to start from a discussion, to see what are your experiences.

Personally I'd not count reactions towards "unread counter" in front of every chat, nor would I send desktop notifications for them...

**Comment by exquo** [comment]
That makes sense - the official signal clients don't send desktop notifications for the received reactions either, nor do they include them in the number of unread messages.

On occasions, reactions can convey enough of a message to warrant a notification (e.g. a confirmation: "Let's meet at X" -- "👍"). But I can see how they can be more of a noise than signal, especially in chats with many participants. 

We can make creating desktop notifications an option (probably disabled by default, to conform to the official clients), e.g. `--notify-on-reactions`. It can apply to the unread count as well (i.e. don't increment them for notifications). We could also try using some kind of a "separate" unread count for just the reactions, but it probably won't be natural for most users.
